app/views/providers/visas/edit.html.erb

```diff
@@ -22,16 +22,19 @@
         <%= page_title %>
       </h1>
 
+      <p class="govuk-body">If you’re unsure if your courses can sponsor visas, check the <a href="https://www.gov.uk/government/publications/register-of-licensed-sponsors-students" class="govuk-link">Register of Student sponsors</a> and <a href="https://www.gov.uk/government/publications/register-of-licensed-sponsors-workers" class="govuk-link">Register of Worker and Temporary Worker licensed sponsors</a>.</p>
```

Comment:
can we just use govuk_link_to here and below?

Reply:
Good call, done.